packages/components/src/experience-control/index.tsx

```diff
@@ -1,97 +1,126 @@
-import { Button } from '@wordpress/components';
+import { BaseControl, useBaseControlProps } from '@wordpress/components';
 import clsx from 'clsx';
+import { useTranslate } from 'i18n-calypso';
 import { IconBad, IconGood, IconNeutral } from './icons';
 
 import './style.scss';
 
-enum Experience {
+export enum ExperienceType {
 	GOOD = 'good',
 	NEUTRAL = 'neutral',
 	BAD = 'bad',
 }
 
+type ExperienceOption = {
+	value: ExperienceType;
+	icon: JSX.Element;
+	ariaLabel: string;
+};
+
+interface ExperienceControlOptionProps {
+	className?: string;
+	checked: boolean;
+	onClick: () => void;
+	children: React.ReactNode;
+	value: string;
+	name: string;
+	ariaLabel: string;
+}
+
 const ExperienceControlOption = ( {
 	className,
-	isSelected,
+	checked,
 	onClick,
 	children,
-}: {
-	className?: string;
-	isSelected: boolean;
-	onClick: () => void;
-	children: React.ReactNode;
-} ) => (
-	<Button
-		className={ clsx( 'experience-control__button', className, {
-			'is-selected': isSelected,
+	value,
+	name,
+	ariaLabel,
+}: ExperienceControlOptionProps ) => (
+	<label
+		className={ clsx( 'experience-control__option', className, {
```

Comment:
We'll likely convert this to CSS Modules soon (#103004), but in the meantime let's add a namespace prefix:

```suggestion
		className={ clsx( 'a8c-experience-control__option', className, {
```